README.md

```diff
@@ -37,6 +37,33 @@ To schedule a daily task, log into the Synology DSM and add a user-defined scrip
           General -> User = root
           Task Settings -> User-defined script = /usr/local/bin/synology-letsencrypt.sh
 
+### Multiple Certificates
+
+If you need to generate more than one certificate with this, you can parameterize synology-letsencrypt.sh with the path of a certificate configuration:
```

Comment:
Wording: remove the "with this", i.e. "If you need to generate more than one certificate ~~with this~~"